packages/app/src/cli/services/dev/processes/graphiql.ts

```diff
@@ -19,13 +21,15 @@ export interface GraphiQLServerProcess extends BaseProcess<GraphiQLServerProcess
 }
 
 export async function setupGraphiQLServerProcess(
-  options: Omit<GraphiQLServerProcessOptions, 'port'>,
+  options: Omit<GraphiQLServerProcessOptions, 'port' | 'shopCustomDomain'>,
```

Comment:
Why are you passing in shopCustomDomain, but then ignoring it afterward?

Reply:
This method was using the same type for the `options` argument and for the `options` filed included in the response. The problem is that the field is mandatory for the response but not for the parameter. 
I guess that something similar was happening with the `port` field, that's why I applied the same strategy